src/region.js

```diff
@@ -204,12 +208,38 @@ Marionette.Region = Marionette.Object.extend({
     return Backbone.$(el, Marionette._getValue(this.options.parentEl, this));
   },
 
+  // Replace the region's DOM element with the view's DOM element.
+  replaceEl: function(view) {
+    // empty el so we don't save any non-destroyed views
+    this.$el.contents().detach();
```

Comment:
A better option would probably be to prevent using both options together. `replaceEl` + `preventDestroy` could throw an error or, at least, should throw a `console.info` that warns the dev about it. I hate the idea of building in a feature that almost encourages poor performance.